REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,292 @@
+Replication policies work quite differently on Android from Java. Please see the relevant section of this guide 
+for [Android](#android-replication-policies) or [Java](#java-replication-policies).
+
+# Android replication policies
+Replication policies on Android run in a `Service` to allow them to run independently of your main application and to allow
+them to be restarted if they are killed by the operating system.  To use replication policies, you need to create a subclass
+one of the existing replication Service components and implement the specifics of your required replications.
+
+##Creating your service
+There are two options when creating your service. 
+
+1. If you don't require periodic replications, you should create a subclass of `ReplicationService`.
+2. If you want your replications to repeat at (roughly) fixed time intervals, as well as possibly adding other criteria that
+are required for replication, you should create a subclass of `PeriodicReplicationService`. This class takes care of the
+scheduling of periodic alarms to trigger the replications. This also requires you to implement a `BroadcastReceiver` that is a subclass of `PeriodicReplicationReceiver`. 
```

Comment:
Assuming `BroadcastReceiver` is an Android class I think a FQN would be useful here. Although the IDE will probably help people end up with the correct class it would be nice to distinguish it from our own library classes.